.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Dependent Issues
+
+on:
+  issues:
+    types:
+      - opened
+      - edited
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+  schedule:
+    - cron: '0 * * * *' # check hourly
+
+jobs:
+  check:
```

Comment:
Can we give this a human-friendly name for the PR UI?

Reply:
example: https://github.com/submariner-io/submariner/blob/devel/.github/workflows/e2e.yml#L9

Reply:
@dfarrell07 see line 22 now, does that seem ok?

Reply:
Much better, thanks @Jaanki!